onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -99,8 +99,9 @@ std::vector<std::unique_ptr<GraphTransformer>> GenerateTransformers(TransformerL
     case TransformerLevel::Level1: {
       std::unordered_set<std::string> l1_execution_providers = {};
 
-      transformers.emplace_back(std::make_unique<ConstantFolding>(l1_execution_providers));
+      transformers.emplace_back(onnxruntime::make_unique<ConstantFolding>(l1_execution_providers));
       transformers.emplace_back(std::make_unique<MatMulAddFusion>(l1_execution_providers));
```

Comment:
transformers.emplace_back(std::make_unique<MatMulAddFusion>(l1_execution_providers));

Reply:
done for all occurrences